crates/transaction-emitter-lib/src/emitter/mod.rs

```diff
@@ -127,13 +125,15 @@ pub struct EmitJobRequest {
     init_gas_price_multiplier: u64,
 
     mint_to_root: bool,
+    skip_minting_accounts: bool,
 
     txn_expiration_time_secs: u64,
     init_expiration_multiplier: f64,
 
     init_retry_interval: Duration,
-
-    max_transactions_per_account: usize,
+    // Either num_accounts or max_transactions_per_account should be set.
+    num_accounts: Option<usize>,
+    max_transactions_per_account: Option<usize>,
```

Comment:
should we have an enum here for these two?

similarly how `mode: EmitJobMode` hides --mempool-backlog and --target-tps ?

Reply:
Good point - that will make it a bit cleaner.